charts/nginz/values.yaml

```diff
@@ -255,6 +255,15 @@ nginx_conf:
     - path: /calls
       envs:
       - all
+    - path: ~* ^/users/([^/]+)/([^/]+)/prekeys/([^/]+)$ 
+      envs:
+      - all
+    - path: ~* ^/users/([^/]+)/([^/]+)/prekeys$
+      envs:
+      - all
```

Comment:
This is not required, everything under /users is already being forwarded to brig.